package org.example.ejercicio_blog.excepciones;public class AllReadyExistsException {
}
